src/database/standalone-sqlite.ts

```
@@ -3448,6 +3479,10 @@ if (!isMainThread) {
           const txIdsMissingOffsets = worker.getTxIdsMissingOffsets(args[0]);
           parentPort?.postMessage(txIdsMissingOffsets);
           break;
+        case 'getTxByOffset':
+          const tx = worker.getTxByOffset(args[0]);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)